website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -122,30 +122,16 @@ base branch.
 To delete a published tag to re-tag, follow these steps:
 
 ```
-    git tag -d vx.y.z
-    git push -d origin vx.y.z
+    git tag -d vx.y.z
+    git push -d origin vx.y.z
 ```
 
 ## Deploying ORCA buckets, RDS cluster and Cumulus ORCA modules via bamboo
 
-For testing purposes, you should use your feature branch in cumulus-orca github repo and [`prototype-latest`](https://ci.earthdata.nasa.gov/browse/ORCA-PL) bamboo plan so that it does not affect the ORCA github `develop` branch. The `prototype-latest` bamboo plan is linked to `feature/ORCA-test-bamboo` github branch as the default branch under cumulus-orca repo. In addition, if changes are made to the bamboo spec file `bamboo.yaml` in this default branch, you have to manually import the bamboo spec by choosing `orca test branch` as the linked repo.
-
-:::warning
-You should reset `feature/ORCA-test-bamboo` before using it.
-1. Rename `feature/ORCA-test-bamboo` to `feature/ORCA-test-bamboo-old`
-1. Create a new branch based off of your branch named `feature/ORCA-test-bamboo`
-1. In the new branch's `bamboo.yaml`:
-   Delete all but one of the plans and the `ORCA-ODP` plan.
-   Change plan's `name` to `prototype-latest`.
-   In each `repositories` element, change `orca-develop` to `orca test branch`.
-   In each `plan` element, change `OI`/`ODP` in `key` values to `PL`
-:::

```

Comment:
I believe these instructions are still valid. As written, `ORCA-test-bamboo` can have changes that are not reflected in `develop`.

Reply:
Added back. I did change "prototype-latest" name to "Test ORCA Deploy Plan" and updated the steps accordingly.

Reply:
Updated.

Reply:
The plan refers to the plan in feature/ORCA-test-bamboo branch and NOT develop branch
https://github.com/nasa/cumulus-orca/blob/feature/ORCA-test-bamboo/bamboo-specs/specs/deploy-cumulus-orca-buckets.yaml#L7

Reply:
The point of these instructions is to run tests against an arbitrary branch. Having to maintain a forked version of our bamboo files with their own modification (which WILL cause conflicts when merging in changes) is not viable.

Reply:
This is why the "You should reset `feature/ORCA-test-bamboo` before using it." are vital. We have to keep the code from the branch under test with as few modifications as possible, and those modifications MUST be documented.

Reply:
Looks like the current develop branch is using ORCA Deploy Plan (ODP) unless I am missing something.

Reply:
Correct. However, the exact strings "ORCA-ODP" and "ORCA-TODP" don't exist in our repository.

Reply:
I believe it is.
I do not understand how the plan develop uses affects whether we need to reset the test branch.